hicommonwealth/commonwealth
# Quill mention users by profile name

## Link to Issue
Closes: #3684

## Description of Changes
- Adds `searchProfiles` endpoint
- Updates quill mention logic to fetch bulk profiles + associated addresses
- Adds client-side cache for quill mention API search requests, to reduce # of API calls
- Updates search bar + search page to fetch profiles instead of addresses

## Test Plan
- Confirm that quill mentions shows expected results for members
- Confirm that search bar shows expected results for members

## Deployment Plan
N/A

## Other Considerations
N/A
